### DISCUSSION TITLE
On knowing about failed login attempts and such

### DISCUSSION THREAD

**Discussion by vlfig-iov42** [concourse/concourse#7190] [Help & Support] [answered]
Warm greetings,

With the intent of keeping a closer eye on an instance that might have to be exposed to the dangers of the open interwebs, I sought to dashboard something that would indicate malicious intent, such as high request rates or 401s.

The request rates I can get from the `concourse_http_responses_duration_seconds_count` but the 401 although theoretically possible for many such routes don't seem to show up for failed logins.

Could it really be true that there's no way to know about failed login attempts?

Thank you

**Comment by taylorsilva** [comment, marked as answer]
Correct, see #5525 for details/discussion
